src/numeric/impl_numeric.rs

```diff
@@ -46,16 +46,24 @@ impl<A, S, D> ArrayBase<S, D>
         sum
     }
 
+    /// Return the sum of all elements in the array.
+    #[deprecated(note="please use `sum` instead", since="0.12.1")]
```

Comment:
Micro comment but can this say it was renamed to `sum`. It's clear it's the same method, just new name.

Reply:
Good point. I've changed it to "renamed to `sum`".